openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/myData/MyDataPageTest.java

```diff
@@ -121,15 +121,15 @@ public void checkHeaders() {
     Events.click(webDriver, By.cssSelector("[data-testid='menu-item-Services']")); // Setting/Services
     Events.click(webDriver, By.cssSelector("[data-testid='whatsnew-modal']")); // What's New
     checkWhatsNew();
-    Events.click(webDriver, By.cssSelector("[data-testid='menu-button'][id='menu-button-Need Help']"));
+    Events.click(webDriver, By.cssSelector("[data-testid='menu-button'][id='menu-button-undefined']"));
     Events.click(webDriver, By.cssSelector("[data-testid='menu-item-Docs']"));
     webDriver.switchTo().window(tabs.get(0));
 
-    Events.click(webDriver, By.cssSelector("[data-testid='menu-button'][id='menu-button-Need Help']"));
+    Events.click(webDriver, By.cssSelector("[data-testid='menu-button'][id='menu-button-undefined']"));
     Events.click(webDriver, By.cssSelector("[data-testid='menu-item-API']"));
     webDriver.navigate().back();
 
-    Events.click(webDriver, By.cssSelector("[data-testid='menu-button'][id='menu-button-Need Help']"));
+    Events.click(webDriver, By.cssSelector("[data-testid='menu-button'][id='menu-button-undefined']"));
```

Comment:
this `undefined` doesn't seem to be correct.
Please check with @Sachin-chaurasiya if this is correct or needs to change on UI side as well to have proper id

Reply:
Changed it back to 'Need Help'. Addressed in the latest commit.

Reply:
@shahsank3t This is not correct so I have shared the patch with @parthp2107 .